cmd/kops/lifecycle_integration_test.go

```diff
@@ -217,6 +219,9 @@ func runLifecycleTest(h *testutils.IntegrationTestHarness, o *LifecycleTestOptio
 				if ownership != expect {
 					t.Errorf("unexpected kubernetes.io/cluster/ tag on %q: actual=%q expected=%q", id, ownership, expect)
 				}
+				if legacy != "" {
+					t.Errorf("added (legacy) KubernetesCluster tag on %q, but it is shared", id)
```

Comment:
Well, shouldn't we do this on route tables?

Reply:
Can you say why you think so?